lib/widgets/issue_item.dart

```diff
@@ -83,7 +83,7 @@ class IssueItem extends StatelessWidget {
                             children: [
                               TextSpan(text: '$title '),
                               TextSpan(
-                                text: '#$number',
+                                text: '#$subtitle',
```

Comment:
Oh, I missed this section. So the `#` is prepended here.

Reply:
How about removing the prefix `#` here, and adding it to `subtitle` props instead?